README.md

```diff
@@ -607,6 +607,17 @@ datasets:
       # For `completion` datsets only, uses the provided field instead of `text` column
       field:
 
+# A list of one or more datasets to eval the model with.
+# You can use either test_datasets, or val_set_size, but not both.
+test_datasets:
```

Comment:
I find it confusing that, in the code base, the term `test_datasets` is used to indicate a dataset meant for evaluation purposes. If I could choose, I would name it `eval_datasets`, also to be consistent with the terminology used by the `DPOTrainer` (see the [docs](https://huggingface.co/docs/trl/main/en/dpo_trainer#trl.DPOTrainer.eval_dataset)).

Similarly, I would also rename `val_set_size` to `eval_set_size` throughout the code.

Thoughts @winglian @younesbelkada?

Reply:
Thanks @filippo82 
I am not 100% familiar with axolotl codebase but I agree that I would lilkely refer to `eval_dataset` instead of `test_dataset` as the first term is the most used terminology for eval dataset for LLMs AFAIK. Also in TRL library we use `eval_dataset` in SFTTrainer init method

Reply:
yeah, let's rename it from test_datasets to eval_datasets

Reply:
Sounds good, but let's split that into a separate PR to keep this one focused.